Sources/SecureXPC/Server/XPCMachServer.swift

```diff
@@ -193,6 +196,19 @@ internal class XPCMachServer: XPCServer {
 		return accept
 	}
 
+    public override var endpoint: XPCServerEndpoint {
+        guard let connection = self.serviceListenerConection else {
+            fatalError("You can only create an `endpoint` for an XPCMachServer after starting it with `start()`.")
```

Comment:
What language version are we targeting? Throwing properties are a really new addition (5.4 or 5.5, IDR).

In any case, I don't think throwing an error is particular useful, because catching the error wouldn't be particularly useful. It's a fundamental misuse of the API which needs to be fixed before shipping.

Reply:
Swift 5.5

But fair point that this is a programming error.